nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-shell-authorizer/src/test/java/org/apache/nifi/authorization/ShellUserGroupProviderBase.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.authorization;
+
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
I think our style guide prohibits or at least frowns on static imports.

Reply:
I've removed those static imports.

Is there a way we can add this to the checkstyle definition?

Reply:
Yes, it should involve [AvoidStarImport](http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport) and [AvoidStaticMemberImport](http://checkstyle.sourceforge.net/config_imports.html#AvoidStaticImport). I can add those to the `nifi-checkstyle.xml` hosted on the wiki, but I will need to check to see if it affects existing code before enforcing it across the project. I can't find at the moment where those rules are documented, but I remember it being enforced semi-rigidly throughout the life of the project.